gradle/ide.gradle

```diff
@@ -21,6 +24,38 @@ allprojects {
   }
 }
 
+tasks.register('configureIdeCheckstyle') {
```

Comment:
can we have proper up-to-date checking for this in case nothing changed?

Reply:
I've defined inputs and outputs, hopefully that's what you were looking for?

Reply:
yes. thx